bpf/lib/fib.h

```diff
@@ -144,7 +145,7 @@ fib_do_redirect(struct __ctx_buff *ctx, const bool needs_l2_check,
 
 			return (int)redirect_neigh(*oif, NULL, 0, 0);
 		} else {
-			union macaddr smac = NATIVE_DEV_MAC_BY_IFINDEX(*oif);
+			union macaddr smac = device_mac(*oif);
```

Comment:
From a visibility point of view, I think it would be preferable to explicitly drop the packet here if we couldn't look up the interface. And not send it out with an all-0 dst MAC.

So maybe this should be a `lookup_device()` instead?

Reply:
Done